### PR TITLE
Fixing NPE while running the build

### PR DESCRIPTION
**Pull request by KurtStam**
Fix for #1351

### CODE REVIEW

**Comment by oscerd** [comment]
Ok to test

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/AutoAdaptableKubernetesClient.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
 import io.fabric8.kubernetes.api.model.PersistentVolume;
 import io.fabric8.kubernetes.api.model.PersistentVolumeList;
 import io.fabric8.kubernetes.api.model.PersistentVolumeClaim;
-import io.fabric8.kubernetes.api.model.PersistentVolumeClaimList;;
+import io.fabric8.kubernetes.api.model.PersistentVolumeClaimList;
```
Comment (rohanKanojia) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/AutoAdaptableKubernetesClient.java]: Why is thi import needed?

Comment (KurtStam) [reply kubernetes-client/src/main/java/io/fabric8/kubernetes/client/AutoAdaptableKubernetesClient.java]: I removed a superfluous semi-colon.

Comment (rohanKanojia) [reply kubernetes-client/src/main/java/io/fabric8/kubernetes/client/AutoAdaptableKubernetesClient.java]: oh, yeah. Thanks ;-)

**Comment by rohanKanojia** [comment]
[merge]

**Pull request merged**